File: Modul-2/Soal 5/PRAK205-2410817120019-NadyaRamadhani.py
#input output pertama
import math
a, b = map(int, input().split())
alas = math.sqrt (b**2 - a**2)
tinggi = a
keliling = tinggi + b + alas
luas = 1/2 * alas * tinggi
print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

print("")

#input output kedua
import math
a = int(input())
b = int(input())
alas = math.sqrt (b**2 - a**2)
tinggi = a
keliling = tinggi + b + alas
luas = 1/2 * alas * tinggi
print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul-2/Soal 5/PRAK205-2410817120019-NadyaRamadhani.c
#include <stdio.h>
#include <math.h>
int main(){
//input output pertama
float a;
float b;
float alas;
float tinggi;
float keliling;
float luas;

printf("");
scanf("%f %f", &a, &b);

alas = sqrt ((b*b) - (a*a));
tinggi = a;
keliling = a + b + alas;
luas = 1.0/2 * alas * tinggi;

printf("Alas = %.0f cm\n", alas);
printf("Tinggi = %.0f cm\n", a);
printf("Keliling = %.0f cm\n", keliling);
printf("Luas = %.0f cm^2\n", luas);

//input output kedua
printf("\n");
printf("");
scanf("%f", &a);
printf("");
scanf("%f", &b);

alas = sqrt ((b*b) - (a*a));
tinggi = a;
keliling = a + b + alas;
luas = 1.0/2 * alas * tinggi;

printf("Alas = %.0f cm\n", alas);
printf("Tinggi = %.0f cm\n", a);
printf("Keliling = %.0f cm\n", keliling);
printf("Luas = %.0f cm^2\n", luas);

return 0;
}